osu.Game/Screens/Select/Leaderboards/LeaderboardScore.cs

```diff
@@ -256,8 +256,23 @@ public LeaderboardScore(Score score, int rank)
             {
                 foreach (Mod mod in Score.Mods)
                 {
-                    // TODO: Get actual mod colours
-                    modsContainer.Add(new ScoreModIcon(mod.Icon, OsuColour.FromHex(@"ffcc22")));
+                    Color4 modColor;
+
+                    switch (mod.Type)

```

Comment:
think about using `Rulesets.UI.ModIcon` since it does exactly the same as this does right now